docs/requirements.txt

```diff
@@ -1,4 +1,3 @@
 -r ../requirements.txt
-sphinx<2.0.0
```

Comment:
We still need sphinx, maybe just without the bound. Probaby works because the theme pulls it?

Reply:
Yes, I was under the impression that before you encountered this problem you hadn't a requirement on Sphinx. If you had, I'll restore the old one.

Reply:
Looking a couple of commits back, I see just ```numpy==1.15.1 dimod==0.7.9``` and don't see a requirement on sphinx in the first three other repos I looked at. So it doesn't seem needed but I can add one if you like.

Reply:
Are you looking at the project requirements, or docs’ requirements? Because sphinx is only required by docs build currently. And yes, we should have it explicitly listed. :)